pkg/extension/workspace/toolchain/registry/types.go

```diff
@@ -14,15 +14,15 @@
 
```

Comment:
wanna kill this directory since it contains another global states outside of `globals` pkg.

Reply:
ok I will delete it in this PR

Reply:
can't as the change is too huge, but we can in a later one